src/main/java/seedu/address/logic/parser/AddClientCommandParser.java

```diff
@@ -42,9 +45,15 @@ public AddClientCommand parse(String args) throws ParseException {
         Phone phone = ParserUtil.parsePhone(argMultimap.getValue(PREFIX_PHONE).get());
         Email email = ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get());
         Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
+        Birthday birthday = null;
+        if (argMultimap.getValue(PREFIX_BIRTHDAY).isPresent()) {
+            String dateString = argMultimap.getValue(PREFIX_BIRTHDAY).get();
+            LocalDate dateLocalDate = ParserUtil.parseDate(dateString);
+            birthday = new Birthday(dateLocalDate);
```

Comment:
so instead of having `new Birthday(dateLocalDate)` here, warp it in a `Optional` and when it is `null` have a empty `Optional`